big-practice/src/js/validation.js

```diff
@@ -14,4 +16,47 @@ const isValidEmail = (value) => (EmailRegex.test(value));
 */
 const isEmpty = (value) => (!value);
 
-export { isValidEmail, isEmpty };
+/**
+ * Validate form
+ *
+ * @param {object} data - The data object contains all the input elements
+ * @param {object} config - Contains information for input checking
```

Comment:
Could you give an example for config? That will be great if we see what we need to pass to config